### ISSUE TITLE
Release schedule

### ISSUE THREAD

**Issue by marcelstoer** [schemaspy/schemaspy#1340]
@jesperolsson-se @npetzall I am eagerly awaiting the release of #1251 / #763. I see that you are regularly committing to master. There's a lot of activity but no release since July 21st. Hence, I went looking for information about your release schedule or milestones.

I noticed that the snapshots are labled as "7.0.0-SNAPHOT" which I take as a hint that the next release is going to be a breaking one. Yet, https://github.com/schemaspy/schemaspy/milestones also shows 6.3.0 and 6.x.x milestones.

-> I'm not quite sure what to make of all of this

Shouldn't each PR be assigned to a milestone? At first glance I didn't see any breaking changes that landed on master since v6.2.4 was released. If that's indeed the case, could you release 6.3 in the near future?

**Comment by npetzall** [comment]
Due to bad continuity from my part both project and milestones are abandoned, need to clean it up.

Breaking changes is a bit harder to explain, depending on which type of interface you're tracking. Arguments, output or even dependencies. I think we are just bundling a few bigger changes and removing some arguments.

But we are moving into the holidays and I'll try to get some time for the project and see what we can do I near time.

**Comment by marcelstoer** [comment]
@jesperolsson-se @npetzall could we maybe get an update on this issue?

**Comment by kayman-mk** [comment]
Yeah, an update would be good. Without reliable release I can't use this tool.

In case the workflows are needed to support automatic releases, I can add a PR for semantic commits and Maven and Docker publish.

**Comment by WadeBarnes** [comment]
Is there any update on when there will be a new release?  Looking for a release that contains the upgrades that address the critical vulnerabilities in spring-boot 2.2.11 contained in the current release.

**Comment by npetzall** [comment]
@marcelstoer we are working on the release notes, it's currently a draft.

**Comment by boblauer** [comment]
Is there a built version of 7.0.2 somewhere that can be downloaded, even if the documentation/release notes are not ready?

**Comment by npetzall** [comment]
https://repo1.maven.org/maven2/org/schemaspy/schemaspy/7.0.2/
